### ISSUE TITLE
Guide the user to doxygen_manual.log when building the manual PDF fails

### ISSUE THREAD

**Issue by ryandesign** [doxygen/doxygen#10740]
**Describe the bug**
Building the manual PDF failed for me:

```
[100%] Generating Doxygen Manual PDF.
cd build/latex && /opt/local/bin/cmake -E remove refman.tex
cd build/latex && /opt/local/bin/cmake -E copy build/doc/manual.sty .
cd build/latex && /opt/local/bin/cmake -E copy doxygen-1.10.0/doc/doxygen_logo.pdf .
cd build/latex && /opt/local/bin/cmake -E chdir build/latex /opt/local/bin/python3.12 doxygen-1.10.0/doc/replace_version.py build/doc/doxygen_manual.tex build/latex/doxygen_manual.tex 1.10.0
cd build/latex && /opt/local/bin/cmake -E chdir build/latex /opt/local/bin/python3.12 doxygen-1.10.0/doc/replace_version.py build/doc/manual.sty build/latex/manual.sty 1.10.0
cd build/latex && /opt/local/bin/pdflatex -shell-escape doxygen_manual.tex
This is pdfTeX, Version 3.141592653-2.6-1.40.25 (TeX Live 2023/MacPorts 2023.66589_5) (preloaded format=pdflatex)
 \write18 enabled.
entering extended mode
(./doxygen_manual.tex
LaTeX2e <2022-11-01> patch level 1
L3 programming layer <2023-02-22>
make[3]: *** [doc/CMakeFiles/doxygen_pdf] Error 1
```

The reason why is not important; I'm sure it's just a missing dependency. The bug I'm reporting is that there is no information in the above output explaining what went wrong or where to go to figure that out. By searching your issue tracker, I found previous similar failures in which the user was asked to consult the file doxygen_manual.log. There is no mention of that file in the above output, nor anywhere in the doxygen source distribution that I can see.

**Expected behavior**
If a build fails and a logfile like doxygen_manual.log is generated whose contains may help the user understand why the build failed, the existence of that logfile should be communicated to the user in the error message.

**Version**
1.10.0

**Comment by albert-github** [comment]
> If a build fails and a logfile like doxygen_manual.log is generated whose contains may help the user understand why the build failed, the existence of that logfile should be communicated to the user in the error message.

Indeed would be good to have a reference to the log file
- in case of building the manual this should probably be done in the doc/CMakeLists.txt
- in case the make command is used the Makefile / make.bat should give this reference.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10741

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
